src/frontend/web/src/ratingpage/RatingPage.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import Slider from 'react-slick';
+import Layout from 'antd/lib/layout';
+
+import 'antd/dist/antd.css';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import './RatingPage.scss';
+import RatingCard from './RatingCard';
+import RatingButton from './RatingButton';
+import FlashAnimation from './FlashAnimation';
+import ArrowIcon from './ArrowIcon';
+
+const leftArrow = 'ArrowLeft';
+const rightArrow = 'ArrowRight';
```

Comment:
I think you should export these two to a more global file. We may want to use them in other places too.